packages/flutter/test_fixes/widgets/drag_target.dart.expect

```diff
@@ -0,0 +1,17 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+void main() {
+  // Changes made in https://github.com/flutter/flutter/pull/133691
+  const dragTarget = DragTarget();
+  dragTarget = DragTarget(onWillAcceptWithDetails: (data) => ());
+  dragTarget = DragTarget(onWillAcceptWithDetails: (data) => ());
+
+  // Changes made in https://github.com/flutter/flutter/pull/133691
+  const dragTarget = DragTarget();
+  dragTarget = DragTarget(onAcceptWithDetails: (data) => ());
+  dragTarget = DragTarget(onAcceptWithDetails: (data) => ());
```

Comment:
Hey @Piinks, even though this resolves the simple renaming issue, I was wondering what might happen when the code is a bit more complex. Let's say that data used to be an 'int' in previous 'onAccept' and the code was something like:

```
onAccept: (data) => {
    myVar += data; 
}
```

Ideally, after switching to `onAcceptWithDetails` this should be changed to:

```
onAcceptWithDetails: (details) => {
    myVar += details.data;
}
```

since details is not an `int` now but a `DragTargetDetails<int>`.

Reply:
No, this is something the user would have to update. Thankfully though they would only need to add the `details.` on the front.